DEVELOPING.md

```diff
@@ -12,7 +12,12 @@ Our development setup assumes a LINUX/BSD environemnt. Windows users may have to
 6. Verify your remotes are properly set up `git remote -v`. You should pull updates from the Box repo `upstream` and push changes to your fork `origin`.
 7. Install dependencies `yarn install`
 8. Test your first build! `yarn run build`
-9. To test the Box UI Elements lauch a local webpack dev server via `yarn run deploy` and navigate to `http://localhost:8080/`. Sample test files are located inside the test folder.
+9. To test the Box UI Elements launch a local webpack dev server via `yarn run dev` and navigate to `http://localhost:8080/`. HTML files for local development are located inside the [`test` folder](http://localhost:8080/test).
+  - Open a test file, such as http://localhost:8080/test/sidebar.html
+  - When prompted, enter a file id and developer token.
+    - Developer tokens can be created at https://cloud.app.box.com/developers/console
```

Comment:
IMO this should link to https://developer.box.com/docs/box-ui-elements#section-using-the-box-ui-elements

Reply:
Added a link at the end. Those docs don't contain step-by-step instructions yet.